app/lib/maat/decision.rb

```diff
@@ -28,11 +28,22 @@ def funding_decision(maat_value)
         maat_value.downcase
       end
 
+      def corrected_ioj_result(response)
```

Comment:
The value updated value of the IoJ decision is not currently being sent. Here we reverse engineer the result.